### ISSUE TITLE
psql: sslmode value "require" invalid when SSL support is not compiled in

### ISSUE THREAD

**Issue by hakunin** [guedes/pgvm#51]
Hi, I get this error message when I try to pull postgresql from Heroku.

Stack overflow says thats a problem with my local psql, that I need to recompile it with SSL support.

How can I achieve this with pgvm?

**Comment by iurev** [comment]
Yes, you may do it if you recompile your version with openssl support.
It worked for me on Manjaro Linux with two libs installed, but probably should work on most systems with openssl lib:

```
core/openssl-1.0 1.0.2.u-1
core/openssl 1.1.1.l-1
```

```
cd ~/.pgvm/src/postgresql-x.y.z
make clean # this is crucial if you have already installed this version before
./configure --with-openssl --without-readline --with-uuid=e2fs --prefix ~/.pgvm/environments/x.y.z/
cd contrib
make
make install
```

I've never done necroposting with such time range before...
I actually doubt anyone still uses this library
